python/cudf/cudf/tests/test_stats.py

```diff
@@ -262,9 +262,6 @@ def test_kurtosis(data, null_flag):
     expected = pdata.kurt()
     np.testing.assert_array_almost_equal(got, expected)
 
-    with pytest.raises(NotImplementedError):
```

Comment:
Instead of removing, can we test now and make sure that it's equal? You should be able to copy the 4 lines above and just add the `numeric_only=True` parameter.

Reply:
Sure! It turns out that Series numeric_only is not implemented in pandas, so my commit reflects that. Now there is the same functionality between cuDF and pandas for series and dataframes as well as tests.

Reply:
And this is why we need to write these tests, without them we would miss so many of these fine distinctions in the pandas API :joy: thanks!